extensions/binding/org.eclipse.smarthome.binding.astro/OSGI-INF/AstroDiscoveryService.xml

```diff
@@ -9,7 +9,7 @@
     http://www.eclipse.org/legal/epl-v10.html
 
 -->
-<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="org.eclipse.smarthome.binding.astro.discovery">
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="false" name="org.eclipse.smarthome.binding.astro.discovery">
```

Comment:
As far as I know, it is always a good practice to make service provider components lazy by setting immediate to false. On the other hand, service consumer components must be eager.